app/jobs/bulk_sync_courses_to_find_job.rb

```diff
@@ -0,0 +1,26 @@
+class BulkSyncCoursesToFindJob < ApplicationJob
+  queue_as :find_sync
+
+  def perform
+    syncable_courses = []
+
+    RecruitmentCycle.current_recruitment_cycle.providers
+      .includes(:latest_published_enrichment)
+      .each do |provider|
+        if provider.publishable?
+          syncable_courses.push(*provider.syncable_courses)
+        end
+      end
```

Comment:
I think this would be clearer as something like (untested code):

```ruby
syncable_courses = RecruitmentCycle.current_recruitment_cycle.providers
                                     .includes(:latest_published_enrichment)
                                     .select(&:publishable?)
                                     .flat_map(&: syncable_courses)
```

Reply:
Ran out off pint and energy...

Reply:
moved to recruitment cycle model